app/views/templates/search/search_object.rb

```diff
@@ -90,19 +89,19 @@ def content
                     } unless  val.blank?
                     res
                   end,
-                  :override_val => 'true',
-                  :overrides => [
+                  override_val: 'true',
+                  overrides: [
                     {
-                      :field_title  => t('site.object.meta-label.ugc'),
-                      :field_url    => root_url + ("search?f[UGC][]=true")
+                      field_title: t('site.object.meta-label.ugc'),
+                      field_url: root_url + ("search?f[UGC][]=true")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.